QuTiP Lecture: Particle emission from a photon cascade

kafischer opened the pull request:
This notebook demonstrates how to calculate the photonic state of the waveguide by integrating the Schrodinger equation for the entire system+waveguide (using quantum stochastic calculus), rather than just correlation functions of the waveguide's state.

ajgpitch commented:
Hi Kevin,
Thanks for this interesting and useful addition.

I have found that this computation
```
P2 = conditionalIteration(g*g.dag(), 2, 0, 0); print("P(2) = " + str(P2))
```
takes a long time (> 10 mins) on my machine. Perhaps having three boxes, one for each of the P0, P1, P2 (with a long time warning on this one) calcs would allow users to skip this is they want to.

Similarly the 'system+channel density matrix' computation takes a long time. Perhaps you could give the user some idea how many dots to expect before 'done'

I guess the string format time dependence does not help before the c_ops change at each iteration and hence the rhs cannot be re-used.

Otherwise, all looks good and I am happy to merge

kafischer commented:
Hi Alex,

Thanks a lot for going through this! I decreased the time resolution by quite a bit so it now runs much faster (~15s for the whole notebook) on my machine without sacrificing too much accuracy, and just removed the dots as a result. I also just added PR#52 in the qutip.github.io repository adding this. When you have a chance would you mind looking that one over as well?

Thanks again!
Kevin

ajgpitch commented:
Happy with this now. Thank you @kafischer  
Merging now
